develop/tutorials/articles/140-assets/rendering-an-asset.markdown

```diff
@@ -33,6 +33,8 @@ to do to get your asset renderer functioning properly for your asset:
 - Create an asset renderer factory to create an instance of the asset renderer
   for each asset entity.
 
+![Figure 1: The asset renderer factory creates an asset renderer for each asset instance.](../../images/asset-renderer-diagram.png)
```

Comment:
Hey @codyhoag 

I am not sure if the multiple arrows between "asset renderer" and "asset instance" are confusing, since the relationship is actually 1 to 1 (1 instance of asset renderer per asset instance)... however, I do see that you only need to write one asset renderer for your assets.... I am not sure, just wanted to let you know it might be confusing... (or maybe it's only me...)